lib/twitter_cldr/js/compiler.rb

```diff
@@ -55,6 +56,34 @@ def compile_each(options = {})
           yield file, TwitterCldr.twitter_locale(locale)
         end
       end
+      def compile_test(options = {})
+        options[:minify] = true unless options.include?(:minify)
```

Comment:
Could you explain this line? I think treating the option as boolean rather than checking if it's present or not is more conventional. If you want an explicit boolean from `nil`, you can do `options[:minify] = !!options[:minify]`, but I don't think even this is necessary here.